### PR TITLE
Docker setup

### PR DESCRIPTION
**Pull request by Camelia-Br**
Create a db container to host a PostgreSQL database

Create a web container to host the Django app

 Install Django, Django REST Framework and psycopg2 as libraries

More details => [Ticket](https://roverdotcom.atlassian.net/jira/software/c/projects/DEV/boards/402?modal=detail&selectedIssue=DEV-82834&assignee=618b962e86c210006a92ca33)

### CODE REVIEW

**Review by ralucafleseriu — APPROVED** [review]
LGTM!

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+version: "3.9"
+services:
+  web:
+    build: .
+    ports:
+      - "5432:5432"
```
Comment (teodorvoicencu) [inline docker-compose.yml]: Preferably I would use the default Django port or one of 8001-8010 if you do not wish to conflict with other ports. Usually the default postgresql configuration is supposed to run on 5432 port

File: django_project/settings.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+]
+
+WSGI_APPLICATION = 'django_project.wsgi.application'
+
+
+# Database
+# https://docs.djangoproject.com/en/3.2/ref/settings/#databases
+
+DATABASES = {
+    "default": {
```
Comment (teodorvoicencu) [inline django_project/settings.py:77]: Just as a side note: you can also configure the database by specifying the connection string. You are not required to use it here but I would like you to also take a look into that. Also please remind me next time to show you how to use environment variables. They are not necesary required for this training but it's a pretty handy way to differentiate between environments such as development, staging and production.

File: requirements.txt [hunk 3]
```diff
@@ -1,4 +1,28 @@
...
-Django==3.2.15
+autopep8==1.6.0
+click==8.1.3
+colorama==0.4.5
+cssbeautifier==1.14.6
+Django==3.2.16
+djangorestframework==3.14.0
+djlint==1.12.3
+EditorConfig==0.12.3
+flake8==4.0.1
```
Comment (teodorvoicencu) [inline requirements.txt:10]: Some of the requirements as this one are not necesary needed when running in a more releasable like environment such as staging or production. Some improvements in the deploy time can be made by removing the dev dependencies from the final requirements.txt that is needed for a deploy on a real environment, especially if there are a lot of packages. Remind me to show you next time how you can separate those.

**Review by teodorvoicencu — APPROVED** [review]

**Pull request merged**